modules/assim.sequential/R/get_ensemble_weights.R

```diff
@@ -23,9 +24,9 @@ get_ensemble_weights <- function(settings){
     ### Loading Weights                                                   ###
     ###-------------------------------------------------------------------###
     weight_file <- read.csv(settings$run$inputs$ensembleweights$path)
```

Comment:
@araiho I'm testing this PR without weights first, but If I wanna test it with weights seems like I need this weights be read in first. Could you plz show me how to make this file ?

Reply:
The weights come from an analysis by John Tipton. There is an example output file from his analysis here: /fs/data2/output/PEcAn_1000008588/ensemble-weights-monte-carlo-mean-harvard.csv